src/openrct2/localisation/Localisation.h

```diff
@@ -131,6 +131,18 @@ class Formatter
     {
         static_assert(sizeof(TSpecified) <= sizeof(uintptr_t), "Type too large");
         static_assert(sizeof(TDeduced) <= sizeof(uintptr_t), "Type too large");
+
+        // checking if the specified type is a valid one
+        // clang-format off
+        static_assert(
+            std::is_same_v<TSpecified, char*>           || std::is_same_v<TSpecified, const char*>  ||
```

Comment:
Hm, maybe put all of them on separate lines? What do you think @tupaschoal ?

Reply:
I definitely prefer one at a line so we don't have to keep aligning the `||`, and also keeping alike types together, sort of this way:
```c++
static_assert(
    std::is_same_v<TSpecified, char*> || 
    std::is_same_v<TSpecified, const char*> ||
    std::is_same_v<TSpecified, int16_t> ||
    std::is_same_v<TSpecified, int32_t> ||
    std::is_same_v<TSpecified, money32> ||
    std::is_same_v<TSpecified, rct_string_id> ||
    std::is_same_v<TSpecified, uint8_t*> ||
    std::is_same_v<TSpecified, uint16_t> ||
    std::is_same_v<TSpecified, uint32_t> ||
    std::is_same_v<TSpecified, utf8*> ||
    std::is_same_v<TSpecified, const utf8*>
);
```